components/smee-client/OWNERS

```diff
@@ -4,3 +4,4 @@ approvers:
 - ifireball
 - gbenhaim
 - amisstea
+- yftacherzog
```

Comment:
Don't you want to add all the team members?

Reply:
I added you as well now. We can add more later.